app.py

```diff
@@ -31,23 +36,72 @@ def healthcheck():
 def login():
     '''
     Login endpoint to authenticate the user
+
+    Expects:
+        JSON body with 'username' and 'password'
     
     Returns:
         JSON response indicating the status of the login
     '''
-    pass
+    data = request.get_json()
+    username = data.get('username')
+    password = data.get('password')
+
+    user = user_model.find_user_by_user(username)
+    if user and user.password == password:
+        app.logger.info('User %s logged in successfully.', username)
+        return make_response(jsonify({'message': 'Login successfully'}), http.HTTPStatus.OK)
+    app.logger.warning('Login failed for username: %s', username)
+    return make_response(jsonify({'error': 'Invalid username or password'}), http.HTTPStatus.UNAUTHORIZED)
 
 @app.route('/auth/create-account', methods=['POST'])
 def register():
     '''
     Register endpoint to create a new user
+
+    Expects: 
+        JSON body with 'username', 'password', and optional 'balance'
     
     Returns:
         JSON response indicating the status of the registration
     '''
-    pass
+    data = request.get_json()
+    username = data.get('username')
+    password = data.get('password')
+    balance = data.get('balance', 100000.0)
+
+    if user_model.find_user_by_username(username):
+        app.logger.warning('Registration failed: username %s already exists.', username)
+        return make_response(jsonify({'error': 'Username already exists'}), http.HTTPStatus.BAD_REQUEST)
+    
+    user = user_model.create_user(username, password, balance)
```

Comment:
it looks like these two weren't included in the commit. can you try to add them again?